main.js

```diff
@@ -0,0 +1,60 @@
+const lecturers = [
+  {
+    fullName: 'Yochai',
+    degree: 'Berkman Professor of Entrepreneurial Legal Studies',
+    details: 'Benkler studies of commons-based peer producation.',
+    image: 'img/lecturers/dave.jpg',
+  },
+  {
+    fullName: 'SohYeong Noh',
+    degree: 'Director of Art centre Nabi and a board member of CC korea',
+    details: 'As the main venue for new media art production in korea.',
+    image: 'img/lecturers/customer-6.jpg',
+  },
+  {
+    fullName: 'Lila Tretikov',
+    degree: 'Executive Director of the Wikimedia foundation',
```

Comment:
- [ ] [OPTIONAL] You might have forgotten something in this section which is behind the author image 

- Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/217827126-61e30813-0355-4734-a53c-f7b7965e19c3.png)

- Your Design 👇 

![image](https://user-images.githubusercontent.com/75126481/217826697-9ebdcf6e-8b3a-4022-8091-fc08c22fec9a.png)